### ISSUE TITLE
mspatcha install fails

### ISSUE THREAD

**Issue by lalloni** [Winetricks/winetricks#2039]
Winetricks fails when trying to install mspatcha:

```
> winetricks mspatcha
Using winetricks 20230212-next - sha256sum: 10c29afe981343a6f132532d102df45ed0294c4685c8ed20669edfb0cd788500 with wine-8.1 and WINEARCH=win32
Executing w_do_call mspatcha
Executing load_mspatcha
grep: warning: stray \ before /
Executing cabextract -q -d /home/plalloni/.wine/dosdevices/c:/windows/temp -L -F i386/mspatcha.dl_ /home/plalloni/.cache/winetricks/win2ksp4/W2KSP4_EN.EXE
/home/plalloni/.wine/dosdevices/c:/windows/temp/i386/mspatcha.dl_: No such file or directory
------------------------------------------------------
warning: Note: command cabextract -q -d /home/plalloni/.wine/dosdevices/c:/windows/temp -L -F i386/mspatcha.dl_ /home/plalloni/.cache/winetricks/win2ksp4/W2KSP4_EN.EXE returned status 1. Aborting.
------------------------------------------------------
```

The issue seems to be that winetricks depended on cabextract to implicitly create parent directories (like the `i386`directory in this case), but it no longer does.

If I try to run cabextract manually like winetricks does:

```
cabextract -q -d /home/plalloni/.wine/dosdevices/c:/windows/temp -L -F i386/mspatcha.dl_ /home/plalloni/.cache/winetricks/win2ksp4/W2KSP4_EN.EXE                                                                         INT ✘
/home/plalloni/.wine/dosdevices/c:/windows/temp/i386/mspatcha.dl_: No such file or directory
```

But if first I create the `i386`directory, then it works:

```
> mkdir /home/plalloni/.wine/dosdevices/c:/windows/temp/i386
> cabextract -q -d /home/plalloni/.wine/dosdevices/c:/windows/temp -L -F i386/mspatcha.dl_ /home/plalloni/.cache/winetricks/win2ksp4/W2KSP4_EN.EXE 
> ls /home/plalloni/.wine/dosdevices/c:/windows/temp/i386                                                                                                                                                                     
mspatcha.dl_
```

**Comment by evren320** [comment]
i have same issue, i suspect cabextract behaviour is recently changed and broke winetricks

**Comment by austin987** [comment]
That's a bug in cabextract (already fixed upstream).

**Comment by evren320** [comment]
thanks, i have installed latest cabextract 1.11 and issue is fixed, previous version was 1.10 which had this issue
